src/Compilers/CSharp/Test/Semantic/Semantics/StaticNullChecking.cs

```diff
@@ -5485,43 +5497,6 @@ public void SimpleWhere()
                 );
         }
 
-        [Fact]
-        public void NotNullWhenFalse_Simple()
-        {
-            CSharpCompilation c = CreateCompilation(@"
-using System.Runtime.CompilerServices;
-public class C
-{
-    public void Main(string? s)
-    {
-        if (MyIsNullOrEmpty(s))
```

Comment:
>if (MyIsNullOrEmpty(s)) [](start = 8, length = 23)

It feels like this scenario is still worth testing.

Reply:
It was moved by additions. It's now on line 6694.